src/editors/containers/ProblemEditor/components/EditProblemView/index.jsx

```diff
@@ -128,6 +132,32 @@ const EditProblemView = ({ returnFunction }) => {
             </Container>
           ) : (
             <span className="flex-grow-1 mb-5">
+              <ProblemEditorPluginSlot
+                updateContent={(newOLX) => {
```

Comment:
[optional]: In this file, I didn’t notice this, but it would be good to introduce a new `updateProblemEditorContent` function and move all logic related to updating the content for `ProblemEditorPluginSlot` into it